_data/taps/extraction/replication-methods/key-based-incremental.yml

```diff
@@ -29,7 +29,7 @@ structural-changes: |
   Detected and automatically handled by Stitch
 
 configuration-requirements: |
-  A `timestamp`, `date-time`, or `integer` column suitable as a [Replication Key]({{ link.replication.rep-keys | prepend: site.baseurl }}).
+  A `TIMESTAMP`, `DATETIME`, or `INTEGER` column suitable as a [Replication Key]({{ link.replication.rep-keys | prepend: site.baseurl }}).
```

Comment:
Oracle uses slightly different sql datatypes and I think this should be `DATE`, `TIMESTAMP`, or `NUMBER`

There are a few different types of `TIMESTAMP`s (`TIMESTAMP WITH TIME ZONE`, `TIMESTAMP WITH LOCAL TIME ZONE`), but they are technically sub-types of `TIMESTAMP`

Reply:
This is a general file that refers to all integrations, not Oracle specifically. I can add `NUMBER` to this.

Reply:
Ok, `NUMBER` does cover `INTEGER`, so if this is general there is no need to change it.  Thanks for the clarification!